web/test/pages/profile/profile-shared.test.tsx

```diff
@@ -357,7 +343,7 @@ describe("profile - shared", () => {
               businessPersona: businessPersona as BusinessPersona,
               municipality: generateMunicipality({ displayName: "Trenton" }),
               industryId: "cannabis",
-              ...getProfileDataMixin(businessPersona as BusinessPersona),
+              ...getForeignNexusProfileFields(businessPersona as BusinessPersona),
```

Comment:
[pebble] We should avoid removing our type-safety by using `as`. If you misspelled "STARTING" or something invalid, this would swallow up and hide the valid compiler error.

You can instead declare types explicitly.

```
const personas: BusinessPersona[] = ["STARTING", "FOREIGN"]
it.each(personas)("....", 
  async (businessPersona: BusinessPersona) => {
```

With this, no `as` is necessary

Reply:
I understand! Replaced it.